app/services/datastore/application_rehydration.rb

```diff
@@ -26,12 +26,20 @@ def already_recreated?
       crime_application.applicant.present?
     end
 
+    def split_case?
+      parent.return_details.reason.inquiry.split_case?
+    end
+
     def applicant
       Applicant.new(parent.applicant.serializable_hash)
     end
 
     def ioj
-      Ioj.new(parent.ioj.serializable_hash) if parent.ioj.present?
+      if parent.ioj.present?
+        Ioj.new(parent.ioj.serializable_hash)
+      elsif split_case?
+        Ioj.new(passport_override: true)
+      end
     end
```

Comment:
can `parent.ioj.present?` and `split_case?` both be false?

Reply:
Yep, possible, in fact is a very likely scenario, but it is covered:

`parent.ioj` will be `nil` if a submitted application has been IoJ passported, as the provider would not have entered any IoJ details then, and this application has been returned for any reason other than split case (like `clarification_required`).

In these scenarios, this `#ioj` method will return `nil` which is the expected behaviour, as we are re-creating the application and the original one didn't have IoJ.